packages/ui/src/components/ConsentToast/ConsentToast.tsx

```diff
@@ -0,0 +1,41 @@
+import { noop } from 'lodash'
+import Link from 'next/link'
+import { Button } from 'ui'
+import { useRouter } from 'next/router'
+
+interface ConsentToastProps {
+  onAccept: () => void
+  onOptOut: () => void
+}
+
+const ConsentToast = ({ onAccept = noop, onOptOut = noop }: ConsentToastProps) => {
+  const { basePath } = useRouter()
+
+  return (
+    <div className="space-y-3">
+      <div className="text-sm">
+        <p>
+          <span className="underline font-medium">No cookies.</span> 🍪
```

Comment:
move className to `<p>`

Reply:
i think this was because we didn't want to underline the cookie

Reply:
ahh okay - lemme simplify the look actually

Reply:
got it!